docs/src/main/sphinx/sql/explain.rst

```diff
@@ -84,6 +84,286 @@ Logical plan::
                               Estimates: {rows: 25 (225B), cpu: 225, memory: 0B, network: 0B}
                               regionkey := tpch:regionkey
 
+EXPLAIN (TYPE LOGICAL, FORMAT JSON)
```

Comment:
We should add a warning indicating that the output is not guaranteed to be stable across Trino versions.